proto/outbound.proto

```diff
@@ -80,18 +80,24 @@ message ProxyProtocol {
     repeated HttpRoute routes = 1;
     // If empty, circuit breaking is not performed.
     FailureAccrual failure_accrual = 2;
+    // If empty, retries will not be enabled for any route.
+    io.linkerd.proxy.destination.RetryBudget retry_budget = 3;
   }
 
   message Http2 {
     repeated HttpRoute routes = 1;
     // If empty, circuit breaking is not performed.
     FailureAccrual failure_accrual = 2;
+    // If empty, retries will not be enabled for any route.
+    io.linkerd.proxy.destination.RetryBudget retry_budget = 3;
   }
 
   message Grpc {
     repeated GrpcRoute routes = 1;
     // If empty, circuit breaking is not performed.
     FailureAccrual failure_accrual = 2;
+    // If empty, retries will not be enabled for any route.
+    io.linkerd.proxy.destination.RetryBudget retry_budget = 3;
```

Comment:
A retry budget is added for GRPC services but we have no RetryPolicy that can be used for grpc services. Is that intentional? Should this be omitted or should a retry policy be added?

Reply:
this can be removed. my intent was to add the budget field now so that it was already there when retry policies for gRPC services are added in the future (presumably as part of GrpcRoute support), but I'm also happy to remove the budget.